conda/plugins/subcommands/doctor/__init__.py

```diff
@@ -10,8 +10,8 @@
 import argparse
 
 from ....base.context import context
-from ....cli.conda_argparse import (
-    ArgumentParser,
+from ....cli.conda_argparse import ArgumentParser
```

Comment:
oops I only noticed now that this is purely for typing purposes so we should just import `ArgumentParser` from `argparse` itself (as we do elsewhere in `conda.cli`)

```suggestion
from argparse import ArgumentParser, Namespace

from ....base.context import context
```